objects/obj_main_menu/Step_0.gml

```diff
@@ -33,19 +33,12 @@ if (stage=2) and (tim1<=0){tim2-=1;
     }
 }
 
-
-// Some kind of integrity test through version? global.version set in Create
-if (global.version<version){
```

Comment:
any integrity tests should be handled during loading saved games. Added a TODO in scr_load, so we won't forget to do that